### ISSUE TITLE
A sound is played everytime I change folder

### ISSUE THREAD

**Issue by raffaem** [flozz/nautilus-terminal#66]
### Problem description

When I change folder in nautilus GUI, a sound is played every time. It didn't happen at all before I installed nautilus-terminal :)

### Installation information

* How did you install (or try to install) Nautilus Terminal (distribution specific package, from sources, what commands you used,...) ?

→ I installed from source, user-wide, latest git commit

* Was it a fresh install or an update?

→ Fresh install

* Have you already installed any older Nautilus Terminal version on the machine before?

→ No

### System information

```
<!-- Please provide the output of the `python3 -m nautilus_terminal --print-debug` command here -->
```

```
$ python3 -m nautilus_terminal --print-debug

NAUTILUS TERMINAL
=================
Version: 4.0.2
System-wide extension: Absent
Current user extension: Installed
Installation path: /run/media/raffaele/55ab61c4-83cf-4d9f-a5cd-7fcfdc14b4fb/progetti_altrui/nautilus-terminal/nautilus_terminal

OPERATING SYSTEM
================
OS: Linux
Platform: Linux-5.12.7-300.fc34.x86_64-x86_64-with-glibc2.33
Version: #1 SMP Wed May 26 12:58:58 UTC 2021
Distribution issue: \S
Kernel \r on an \m (\l)

PYTHON
======
Python version: 3.9.5

SYSTEM DEPENDENCIES
===================
Nautilus Python: Installed
GLib schemas compiler: Installed
```

**Comment by flozz** [comment]
The terminal bell is maybe activated when the terminal is cleared or when the command is cleaned before entering the `cd` command...

Maybe you can try some of the following solutions to disable it?

* https://www.cyberciti.biz/faq/how-to-linux-disable-or-turn-off-beep-sound-for-terminal/

Also you may try to add the following line:

```python
self._ui_terminal.set_audible_bell(False)
```

here:

 https://github.com/flozz/nautilus-terminal/blob/master/nautilus_terminal/nautilus_terminal.py#L285

**Comment by EtienneMueller** [comment]
@raffaem I had the same problem. Simply deactivating auto-cut-user-input in the settings solved the problem for me.

**Comment by raffaem** [comment]
> @raffaem I had the same problem. Simply deactivating auto-cut-user-input in the settings solved the problem for me.

Hi.

What do you mean? What are the exact steps to follow to deactivate that option?

It is still happening with Fedora 35 Workstation

**Comment by raffaem** [comment]
> The terminal bell is maybe activated when the terminal is cleared or when the command is cleaned before entering the `cd` command...
> 
> Maybe you can try some of the following solutions to disable it?
> 
>     * https://www.cyberciti.biz/faq/how-to-linux-disable-or-turn-off-beep-sound-for-terminal/

Disabling the bell from a separate instance of GNOME Terminal does not fix the issue

**Comment by raffaem** [comment]
> Also you may try to add the following line:
> 
> ```python
> self._ui_terminal.set_audible_bell(False)
> ```
> 
> here:
> 
> https://github.com/flozz/nautilus-terminal/blob/master/nautilus_terminal/nautilus_terminal.py#L285

This fixes the issue.

That line should be added by default.

**Comment by flozz** [comment]
→ Fix released in **Nautilus Terminal v4.0.4** :)
